### ISSUE TITLE
Tester issue report for: "Navigate to menubar"

### ISSUE THREAD

**Issue by robfentress** [w3c/aria-at#251]
### Test file at exact commit

[tests/menubar-editor/test-05-navigate-to-menubar-interaction.html](https://github.com/w3c/aria-at/blob/2a3c46f1cbd20e7f2f0a706052d2d82aaf207a29/tests/menubar-editor/test-05-navigate-to-menubar-interaction.html)

### Cycle:

Test Pilot (2020-05-27)

### AT:

VoiceOver for macOS (version macOs 10.15.4)

### Browser:

Chrome (version 83.0.4103.61)

### Description

The number of items was announced, but not the position is the set.  Therefore I marked "The position of the item and number of items in the menubar are conveyed" as “Incorrect output”.  These should probably be broken out as separate assertions.

**Comment by mcking65** [comment]
Yes, the need to be separated! We have a problem in the system ... for some reason the number of assertions is capped at 5. Based on this situation, I think fixing this is critical because this should be one good output and one no output, which is way better than incorrect output.

Related to #193.
